system-tests/src/test/java/io/apicurio/registry/systemtests/TestBase.java

```diff
@@ -40,6 +45,44 @@ public TestBase() {
         setupTestClass();
     }
 
+    @BeforeAll
+    protected void beforeAllTests() throws InterruptedException {
```

Comment:
The mentioned `kubernetes-client` JUnit extension might help here and reduce the code we have to maintain on our side.